frame/scheduler/src/lib.rs

```diff
@@ -15,38 +15,63 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//! # Scheduler
-//! A Pallet for scheduling dispatches.
+//! # Scheduler Pallet
 //!
-//! - [`Config`]
-//! - [`Call`]
-//! - [`Pallet`]
+//! A Pallet for scheduling runtime calls.
 //!
 //! ## Overview
 //!
-//! This Pallet exposes capabilities for scheduling dispatches to occur at a
-//! specified block number or at a specified period. These scheduled dispatches
+//! This Pallet exposes capabilities for scheduling runtime calls to occur at a
+//! specified block number or at a specified period. These scheduled runtime calls
 //! may be named or anonymous and may be canceled.
 //!
-//! **NOTE:** The scheduled calls will be dispatched with the default filter
+//! **NOTE:** The scheduled runtime calls will be dispatched with the default filter
 //! for the origin: namely `frame_system::Config::BaseCallFilter` for all origin
 //! except root which will get no filter. And not the filter contained in origin
 //! use to call `fn schedule`.
```

Comment:
```suggestion
//! **NOTE:** Instead of the filter contained in the origin used to call `fn schedule`, scheduled runtime calls will be 
//! dispatched with the default filter for the origin: namely `frame_system::Config::BaseCallFilter` for all origin
//! types (except root which will get no filter).
```